packages/client/tests/functional/issues/8832/tests.ts

```diff
@@ -117,61 +118,69 @@ testMatrix.setupTestSuite(
        * in the query filters.
        */
       describe('unhandled filters', () => {
-        test('should fail with `Assertion violation` when "in" is repeated at least twice and "n" is the default value of $QUERY_BATCH_SIZE', async () => {
-          expect.assertions(3)
-          const n = 32766
-          const ids = await createTags(n)
-
-          try {
-            await prisma.tag.findMany({
-              where: {
-                OR: [
-                  {
-                    id: { in: ids },
-                  },
-                  {
-                    id: { in: ids },
-                  },
-                ],
-              },
-            })
-          } catch (e) {
-            const error = e as Prisma.PrismaClientKnownRequestError
-            expect(error.message).toContain(
-              'Assertion violation on the database: `too many bind variables in prepared statement, expected maximum of 32767, received 65533`',
-            )
-            expect(error.code).toBe('P2035')
-            expect(error.meta).toMatchObject({
-              database_error:
-                'too many bind variables in prepared statement, expected maximum of 32767, received 65533',
-            })
-          }
-        })
-
-        test('should fail with `Assertion violation` when "in" has 32766 ids and a "take" filter', async () => {
-          expect.assertions(3)
-          const n = 32766
-          const ids = await createTags(n)
-
-          try {
-            await prisma.tag.findMany({
-              where: {
-                id: { in: ids },
-              },
-              take: 1,
-            })
-          } catch (e) {
-            const error = e as Prisma.PrismaClientKnownRequestError
-            expect(error.message).toContain(
-              'Assertion violation on the database: `too many bind variables in prepared statement, expected maximum of 32767, received 32768`',
-            )
-            expect(error.code).toBe('P2035')
-            expect(error.meta).toMatchObject({
-              database_error:
-                'too many bind variables in prepared statement, expected maximum of 32767, received 32768',
-            })
-          }
-        })
+        // TODO this test does not fail as expected
```

Comment:
Actually, it's expected that this does not fail since the failure is only expected when using a prepared statement.
Right?

Reply:
I think that's something I wrote down somewhere, basically asking about prepared statements and if we would support them?

Reply:
Driver Adapters are currently not checking `QUERY_BATCH_SIZE` at all. It looks like [this code](https://github.com/prisma/prisma-engines/blob/af46365b40aa4864b99acec97dc2dd5bdc27411c/query-engine/connectors/sql-query-connector/src/context.rs) was only called on Quaint. It's unclear at the moment whether that could result in bugs or not.